.github/workflows/ci.yml

```diff
@@ -295,10 +218,9 @@ jobs:
         docs,
         test-sqlite,
         test-postgres,
-        test-mysql-mariadb,
-        test-mssql,
+        test-oldest-latest
       ]
-    if: github.event_name == 'push' && (github.ref == 'refs/heads/v6' || github.ref == 'refs/heads/v7')
+    if: github.event_name == 'push'
```

Comment:
isn't this going to cause pushes on `main` to publish?

Reply:
CI is not running on `main` so I hope not